lib/solidus_retail/order/generate_pos_order.rb

```diff
@@ -45,17 +45,37 @@ def create_order
 
```

Comment:
I can totally move my code into App/ . It's generally a 'best' practice to avoid this because it's much quicker to test. However, it's only a lazy developer best practice because we are attempting to speed up the red/green/refactor loop that comes with TDD